rust/noosphere-core/src/view/mutation.rs

```diff
@@ -47,20 +50,20 @@ impl<S: BlockStore> SphereRevision<S> {
 #[derive(Debug)]
 pub struct SphereMutation {
     did: String,
-    links: LinksMutation,
-    names: NamesMutation,
-    allowed_ucans: AllowedUcansMutation,
-    revoked_ucans: RevokedUcansMutation,
+    content: ContentMutation,
+    identities: IdentitiesMutation,
+    delegations: DelegationsMutation,
+    revocations: RevocationsMutation,
```

Comment:
Simple, and given the data spheres store, non-ambiguous, especially compared to "links"/"names" -- huge improvement in clarity here IMO